Benqi-Agents/Large-Grant/src/finding.ts

```diff
@@ -0,0 +1,24 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import { ThresholdMode } from "./utils";
+
+export function createFinding(
+  recipient: string,
+  amount: string,
+  thresholdMode: ThresholdMode,
+  threshold: string
+): Finding {
+  return Finding.fromObject({
+    name: "Large QI Grant",
+    description: "There was a large QI Grant in the BENQI Comptroller contract",
+    alertId: "BENQI-4",
+    protocol: "Benqi Finance",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    metadata: {
+      recipient,
+      amount: amount.toString(),
+      mode: ThresholdMode[thresholdMode],
```

Comment:
I don't think this field is needed.
The agent will be deployed with one `mode` ( chosen by the client), so that field will remain the same for all generated findings. Therefore, it isn't bringing any useful Information related to the detected event.

Reply:
I thought this might be useful if they deploy multiple versions of the agent, but I think this might not be a common use case after all.

Reply:
Removed in [3e4e236](https://github.com/NethermindEth/Forta-Agents/pull/183/commits/3e4e2366dabeb2bf5c8baa3c6f1d4c47d65ea40d).